### PR TITLE
[doc] Add python docstring for gui.py

### PR DESCRIPTION
**Pull request by Leonz5288**
Related issue = #

### CODE REVIEW

File: python/taichi/misc/gui.py [hunk 1]
```diff
@@ -149,11 +207,25 @@ def cook_image(self, img):
         return np.ascontiguousarray(img)
 
     def get_image(self):
+        """Get the image data.
+
+        Returns:
+            The image data in numpy ascontiguousarray type.
```
Comment (ljcc0930) [inline python/taichi/misc/gui.py:213]:
```suggestion
            The image data in numpy contiguous array type.
```

File: python/taichi/misc/gui.py [hunk 2]
```diff
@@ -102,21 +121,60 @@ def value(self, value):
...
+        """Create a slider object on canvas to be manipulated with.
+
+        Args:
+            text (str): The title of slider.
+            minimum (Number): The minimum value of slider.
+            maximum (Number): The maximum value of slider.
+            step (Number, optional): The changing step of slider. Default is 1.
+
+        Return:
+            The created slider object.
```
Comment (ljcc0930) [inline python/taichi/misc/gui.py:133]: Add the type of return value?

e.g.:
```suggestion
        :class:`~taichi.misc.gui.GUI.WidgetValue` : The created slider object.
```

Comment (Leonz5288) [reply python/taichi/misc/gui.py:133]: Got it. I'll work on this.

**Review by k-ye — APPROVED** [review]
Awesome!

**Pull request merged**